requirements/edx/base.txt

```diff
@@ -744,7 +744,7 @@ oauthlib==3.0.1
     #   social-auth-core
 openedx-calc==3.0.1
     # via -r requirements/edx/base.in
-openedx-events==0.11.1
```

Comment:
Maybe we should let the event-bus folks we're updating this, since the 0.12.0 has a breaking change.

Reply:
Thanks for the update.

Reply:
Perfect then! thank you both.